modules/assim.batch/R/pda.load.data.R

```diff
@@ -81,6 +89,9 @@ load.pda.data <- function(settings, con) {
       
       inputs[[i]]$obs <- AMFo
       inputs[[i]]$par <- c(AMF.params$intercept, AMF.params$slopeP, AMF.params$slopeN)
+    }else{
+      inputs[[i]]$obs <- inputs[[i]]$data[colnames(inputs[[i]]$data) %in% inputs[[i]]$variable.name]
+      inputs[[i]]$par <- sd(unlist(inputs[[i]]$obs), na.rm = TRUE) # testing
```

Comment:
1) seems like an OK interim solution, but we'll want to generalize
2) unfortunately yes. The data itself is not consistent between the chambers and the tower, but I believe the soil resp data provides important info on the temperature (and possibly moisture) sensitivity of Rsoil
3) So the fit right now is too biased to assess the residuals, but if you imagine the multiplicative bias term added then Gaussian might not  be that bad. Need to reassess once that's in there with standard residual and QQ plots